bed/methods.go

```diff
@@ -84,6 +103,8 @@ type ByGenomicCoordinates struct {
 	BedSlice
 }
 
+// Less sorts by genomic coordinate and satisfies the sort.Slice
```

Comment:
May be helpful to stay that it sorts by chromosome, and then entries within the same chromosome are sorted from smallest to largest by genomic coordinates (first by start coordinate and then ties are broken by the end coordinate).